### PR TITLE
Run changelog check only on PRs, add support for no-changelog-needed …

### PR DESCRIPTION
**Pull request by maxnoe**
…label

### CODE REVIEW

**Review by LukasNickel — APPROVED** [review]

**Review by StFroese — APPROVED** [review]

File: .github/workflows/changelog.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+jobs:
+  changelog:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Check for news fragment
+        if: ${{ ! contains( github.event.pull_request.labels.*.name, 'no-changelog-needed')}}
```
Comment (StFroese) [inline .github/workflows/changelog.yml:20]: This seems to not work. Check still fails in this PR

Comment (maxnoe) [reply .github/workflows/changelog.yml:20]: ? All ten checks passed, including their changelog check

Comment (StFroese) [reply .github/workflows/changelog.yml:20]: Ok, I see. That’s weird I re-ran it two times after you added the label and they both failed. Maybe it was just a bug in the GitHub app…

Comment (StFroese) [reply .github/workflows/changelog.yml:20]: image

Comment (StFroese) [reply .github/workflows/changelog.yml:20]: Definitely a bug with the app

Comment (maxnoe) [reply .github/workflows/changelog.yml:20]: 1 failed, 10 passed? But there are only 10 checks...

**Pull request merged**